runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -2412,15 +2406,17 @@ J9::Z::TreeEvaluator::inlineCRC32CUpdateBytes(TR::Node *node, TR::CodeGenerator
    cg->stopUsingRegister(end);
    cg->stopUsingRegister(remaining);
 
-   // Dependencies common between main line and OOL
-   dependencies = new (cg->trHeapMemory()) TR::RegisterDependencyConditions(0, 5, cg);
+   // Dependencies for outer ICF
```

Comment:
Minor nit --as there's only one ICF now, no need to say "outer" ICF.

Reply:
I'll just get rid of the comment altogether. Now that there is only one bunch of dependencies I probably don't need to specify which one I'm using.